softvis3d-frontend/checkTestFileLocationAndName.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
This was just a quick approach to do this. In the end, it would be great to run this as part of the build process, so javascript / typescript would be better here.